functions/Get-DbaPrivilege.ps1

```diff
@@ -51,17 +51,19 @@ function Get-DbaPrivilege
     [PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential
   )
 
-BEGIN
+  BEGIN
   {
-    function Convert-SIDToUserName ([string] $SID ) {
-      $objSID = New-Object System.Security.Principal.SecurityIdentifier ("$SID") 
-      $objUser = $objSID.Translate( [System.Security.Principal.NTAccount]) 
-      $objUser.Value
+    $ResolveSID = @"
```

Comment:
Hey Klaas, Is there a reason this has to be in text? Could it be either be a [scriptblock]::Create(, or { } or just passing it whole to Invoke-Command even if it's duplicated code?